native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -46,9 +47,9 @@ const styles = StyleSheet.create({
     backgroundColor: colorWhite,
     elevation: 3,
     shadowColor: colorGray900,
-    shadowOffset: { width: 0, height: 1 },
+    shadowOffset: { width: 0, height: 1 / PixelRatio.get() },
```

Comment:
Hmm so this will produce half points. Interesting, but if it does what we want I guess 🤷‍♂️